CMakeLists.txt

```diff
@@ -197,19 +197,11 @@ endif(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le")
 
 option(PORTABLE "build a portable binary" OFF)
 option(FORCE_SSE42 "force building with SSE4.2, even when PORTABLE=ON" OFF)
-if(PORTABLE)
-  # MSVC does not need a separate compiler flag to enable SSE4.2; if nmmintrin.h
-  # is available, it is available by default.
-  if(FORCE_SSE42 AND NOT MSVC)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse4.2 -mpclmul")
```

Comment:
I think we should still set the flags when `PORTABLE && FORCE_SSE42`?

Reply:
When `FORCE_SSE42=ON`, either `HAVE_SSE42` is true and the flags will be set properly, or we will print a warning and exit. So it seems nothing to do with `PORTABLE`?

Reply:
It seems to me the logic here is "if force_sse42 set sse42 flags for all files (not just crc32c.cc)". Am I reading it wrongly? I'm not familiar with CMake.